46.md

```diff
@@ -25,7 +25,7 @@ This is most common in a situation where you have your own nsecbunker or other t
 The remote signer would provide a connection token in the form:
 
 ```
-bunker://<remote-pubkey>?relay=<wss://relay-to-connect-on>&relay=<wss://another-relay-to-connect-on>&secret=<optional-secret-value>
+bunker://<remote-pubkey>?relay=<wss://relay-to-connect-on>&relay=<wss://another-relay-to-connect-on>&secret=<optional-secret-value>&clientkey=<optional-hex-key-to-use-as-the-clients-key>
```

Comment:
I would go with `sk` or `secretKey` personally. I think it's more important that it's secret than which side of the flow it's on.

```suggestion
bunker://<remote-pubkey>?relay=<wss://relay-to-connect-on>&relay=<wss://another-relay-to-connect-on>&secret=<optional-secret-value>&sk=<optional-hex-key-to-use-as-the-clients-key>
```

Reply:
I just thought that `clientkey` reduces the chances of someone actually writing the main user secret key in there.

Reply:
Fair enough.